src/app/work-item/work-item-list/work-item-list.component.scss

```diff
@@ -44,7 +92,7 @@
   cursor: pointer;
 }
 
-@media (max-width: ($grid-float-breakpoint - 1)){
+@media (max-width: ($grid-float-breakpoint - 1)){  
   .containerPadd-nologin{ padding-bottom: 0; }
   .workItem_Overlay{
     background-color:$color-pf-black;
```

Comment:
This change is not related to this PR. We need to raise another issue to fix this.